fendermint/eth/api/src/state.rs

```diff
@@ -442,6 +443,12 @@ where
         height: FvmQueryHeight,
     ) -> JsonRpcResult<ActorType> {
         let addr = to_fvm_address(*address);
+
+        if let Some(actor_type) = self.addr_cache.get_actor_type_from_addr(&addr) {
+            log::debug!("cache contains the result: {:?}, return it", actor_type);
```

Comment:
```suggestion
            tracing::debug!(addr, actor_type, "cache hit");
```
I'm not sure if there's a difference between using `tracing::log::debug!` but usually we use `tracing::debug!`. The macro supports structured logging like `debug!(primitive_value, key2 %= value_with_display, ?value_with_debug, "static message")` which means those values will turn up in a way that is easier to filter for in log aggregators. We should prefer them to interpolating values into the message.